.devcontainer/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# syntax=docker/dockerfile:1.5
+
+ARG BASE
+ARG PYTHON_PACKAGE_MANAGER=conda
+
+FROM ${BASE} as pip-base
+
+ENV DEFAULT_VIRTUAL_ENV=rapids
+ENV RAPIDS_LIBUCX_PREFER_SYSTEM_LIBRARY=true
+
+RUN apt update -y \
+ && DEBIAN_FRONTEND=noninteractive apt install -y \
+    libblas-dev liblapack-dev \
+ && rm -rf /tmp/* /var/tmp/* /var/cache/apt/* /var/lib/apt/lists/*;
+
```

Comment:
This can probably be removed too.
```suggestion
```

RMM is a more minimal example: https://github.com/rapidsai/rmm/blob/branch-25.04/.devcontainer/Dockerfile

Reply:
Yep agree!

After this change, the only difference between this dockerfile and that `rmm` one would be:

```dockerfile
ENV RAPIDS_LIBUCX_PREFER_SYSTEM_LIBRARY=true
```

And that really is needed. It was added across devcontainers for projects relying on `libucx` wheels in https://github.com/rapidsai/build-planning/issues/118

Reply:
fixed in https://github.com/rapidsai/ucxx/pull/363/commits/3c774c2314d6fee7d3b68440af17b838b439d989